### DISCUSSION TITLE
MPI10k & JKBMS commands

### DISCUSSION THREAD

**Discussion by michaelm75** [jblance/mpp-solar#285] [Q&A] [answered]
Hello,
Is MPI 10K supported?
Another question, is it possible to control the charge/ discharge command for the JKBMS? Would be useful to avoid unnecessary standby consumption when PV is not charging and amount of energy is low.
Thanks

**Comment by jblance** [comment, marked as answer]
i think i have seen it used
try connecting and trying to get the inverter to respond to a protocol id query
jkbms control is limited, i dont have that control yet, though it is possible to add it

**Reply by michaelm75** [reply]
Thank you

**Comment by exilaus** [comment]
possible add setDischargingOn  setDischargingOff  on JKv11 protocoll too?
